src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.bigSearchBar::placeholder,
+.smallSearchBar::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+}
+
+.smallSearchBar::placeholder {
+  font-size: 14px;
+}
+
+.search-form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.bigSearchBar,
+.smallSearchBar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61,78,97,0.1);
+  border-radius: 4px;
+  height: 70px;
+  padding-left: 62px;
+}
+
+.smallSearchBar {
+  padding-bottom: 2px;
+  height: 42px;
+  padding-left: 33px;
+}
+
+.bigLoop,
+.smallLoop {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+}
+
+.smallLoop {
+  left: 13px;
+  top: 15px;
+  height: 11px;
+  width: 11px;
+}
+
+.bigSearchBar:hover,
+.smallSearchBar:hover {
+  box-shadow: 0 3px 8px rgba(61,78,97,0.2);
+}
+
+.bigSearchBar:focus,
+.smallSearchBar:focus {
+  background: linear-gradient(180deg,#fff,#f6f6f7);
+  border-radius: 3px;
+  color: #3d4e61;
+  font-weight: 700;
+}
```

Comment:
Again, if you'll apply two classes to small input, there will be no need to duplicate this rules